tensorboard/plugins/interactive_inference/witwidget/version.py

```diff
@@ -14,4 +14,4 @@
 
 """Contains the version string."""
 
-VERSION = '0.1.6a1'
+VERSION = '0.1.6a4'
```

Comment:
No action required: Are you actively pushing to pip or something? I am just surprised that you are updating this

Reply:
Yes pushing to pip to get these fixes out to notebook-mode users.